### DISCUSSION TITLE
There is no Archives section in Settings

### DISCUSSION THREAD

**Discussion by Pierre-Roux** [community/community#133945] [Projects and Issues] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

There is no Archives section in Settings
I want to Include Git LFS objects in archives. But there's no archives

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by samus-aran** [comment, marked as answer]
Hi @Pierre-Roux 👋 

Check out the GitHub documentation for [Managing Git LFS objects in archives of your repository](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-git-lfs-objects-in-archives-of-your-repository) but I suggest you read the GitHub Archive Program and if you are opted in/out and its [FAQs.](https://archiveprogram.github.com/faq/)
